python3x/GET

```diff
@@ -55,6 +55,9 @@ then
 	echo "ERROR some libraries required by python might be missing"
 	exit 1
 fi
+
+#replace sqlite with a non-MREMAP version
+rsync -a $BASEDIR/../sqlite/libsqlite3.so.0 $ROOTFS/usr/lib/libsqlite3.so.0
```

Comment:
nitpick: why "rsync" here instead of just "cp"? It's possible that some users will not have rsync installed, so why rely on it if we don't need to?

Reply:
`python3x` won't build without `rsync` anyway. `python3x/GET` currently uses it for all FS operations, to avoid copying files four times due to how `install_shlibs` is called. I just stuck to the convention. Changing to `cp` is fine by me, since this part of `main` is only called once